tests/kuttl/tests/ovn_restart_flow/03-assert.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Check for:
+#
+# - The current bridges flows are present in the logs; these flows writen in
+#   the logs are the ones restored from the previous "ovs-vswitchd" container
+#   execution.
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 300
+commands:
+    - script: |
+        controller_pod=$(oc get pod -n $NAMESPACE -l service=ovn-controller-ovs -o name|head -1)
+        logs=$(oc logs -n $NAMESPACE -f $controller_pod -c ovs-vswitchd --follow=false)
+        echo "Logs from the controller pod and ovs-vswitchd container:"
+        echo "$logs"
+        flows=$(oc exec -n $NAMESPACE --container ovs-vswitchd $controller_pod -- bash -c 'for br in `ovs-vsctl list-br`; do ovs-ofctl dump-flows $br --no-stats --bundle; done')
+        echo "OVS flows:"
+        echo "$flows"
+        oc exec -n $NAMESPACE --container ovs-vswitchd $controller_pod ovs-vsc
+        for flow in "${flows[@]}"; do
+            # We exclude the tunnel ports from this check.
```

Comment:
Could you please explain why these flows are special? (In this comment.)

Reply:
Because the ovn-controllers were spawning at a different rate and creating the tunnel ports in different test steps, messing with the flow table (1st step: just one tunnel port, 2nd step: two ports created --> flow table different).

Reply:
This particular bit i think should have been resolved with https://github.com/openstack-k8s-operators/ovn-operator/pull/311 , anyway we don't need to do that with test bridge approach.